src/nls/pl/strings.js

```diff
@@ -129,6 +129,7 @@ define({
     "SAVE_CLOSE_MULTI_MESSAGE"          : "Czy chcesz zapisać zmiany w następujących plikach?",
     "EXT_MODIFIED_TITLE"                : "Zmiany zewnętrzne",
     "CONFIRM_DELETE_TITLE"              : "Potwierdź usunięcie",
+    "CONFIRM_FILE_DELETE"               : "Czy na pewno chcesz usunąć plik <span class='dialog-filename'>{0}</span>?",
     "CONFIRM_FOLDER_DELETE"             : "Czy na pewno chcesz usunąć folder <span class='dialog-filename'>{0}</span>?",
```

Comment:
I tried to fit my translation to the previous version, hence similar strings. I think "czy na pewno chcesz usunąć(...)" could stay unchanged because it is a common phrase in different UI (e.g. MS Windows 7 -> "Czy na pewno chcesz przenieść ten plik do kosza?") and look normal, but in fact for me it's a trifle.